src/box/lua/key_def.c

```diff
@@ -62,6 +62,9 @@ luaT_push_key_def(struct lua_State *L, const struct key_def *key_def)
 		lua_pushboolean(L, key_part_is_nullable(part));
 		lua_setfield(L, -2, "is_nullable");
 
+		lua_pushboolean(L, part->exclude_null);
+		lua_setfield(L, -2, "exclude_null");
```

Comment:
BTW, can we don't push the `exclude_null` field when it has the default value? I see, you follows  the behaviour around `is_nullable`, but it looks as the bad pattern for me. We don't show default collation or JSON path. We don't show default `is_nullable = false` in a space format serialization. However we show it in the index parts serialization (and I don't know why).

@alyapunov What do you think?

Reply:
> it looks as the bad pattern for me
I agree. If we also remove default `is_nullable = false` from index - I think it will be perfect.
PS I also thought about it but `is_nullable` implementation confused me.